### DISCUSSION TITLE
How create SEO Friendly URL on Orchard.Core?

### DISCUSSION THREAD

**Discussion by Diman96** [OrchardCMS/OrchardCore#9745] [Q&A]
What is the best way to do things like slash control and lower case letters in url to redirect?
Exemple `https://mysite.com/////Ru/Products -> https://mysite.com/ru/products`

I didn't find any mention of this topic in the documentation https://docs.orchardcore.net/en/dev/ 

I found a solution to the problem for Asp.NetCore, but I'm not sure if it will work correctly for Orchard
https://github.com/Dotnet-Boxed/Framework
 
Can you tell me how best to solve this problem?

**Comment by hishamco** [comment]
@agriffard is our SEO module doing that or just keywords and metadata?

**Reply by Diman96** [reply]
@agriffard @hishamco  
Some information has appeared on this topic?

**Reply by xperiandri** [reply]
We also need a configurable ability to set multiple URLs for content item. Where this alternate URLs will permanently redirect to master URL of AutoRoute

**Comment by xperiandri** [comment]
Also there are such recommendation available for ASP.NET Core in general https://stackoverflow.com/a/53838799/1149328
Should we follow that?
